spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -323,31 +334,40 @@ def reset_icon_provider(self):
 
     #---- Context menu
     def setup_common_actions(self):
-        """Setup context menu common actions"""
+        """Set-up context menu common actions."""
+        # Show all files
+        self.all_action = create_action(self, _("Show all files"),
+                                        toggled=self.toggle_all)

```

Comment:
Please format this code in the same way as you did with `self.hidden_action` below.